app/src/theme/index.tsx

```diff
@@ -1,3 +1,34 @@
+import React from 'react'
+import { ThemeProvider as StyledThemeProvider } from 'styled-components'
+
+import { Colors } from './types'
+
+export function colors(darkMode: boolean): Colors {
+  return {
+    // text
+    text1: darkMode ? '' : '#37474F',
+    text2: darkMode ? '#C3C5CB' : '#86909E',
+    text3: darkMode ? '#6C7284' : '#86909E',
```

Comment:
@Mi-Lan  text2 and text3 have the same value on the code, mistake? figma has a different value for text3
![Screenshot 2021-06-22 at 16 16 08](https://user-images.githubusercontent.com/57708955/122941104-5e338080-d375-11eb-8ed7-da787dd27e89.png)